docs/playroom/snippets.js

```diff
@@ -742,10 +742,9 @@ items={[
         <SummaryListItem>
           <SummaryListItemTerm>Contact information</SummaryListItemTerm>
           <SummaryListItemDescription>
-            <Stack>
```

Comment:
In all of our examples, we can just use a simple `br` tag instead of a `Stack` with `Text` elements.